src/pudl/etl/eia_api_assets.py

```diff
@@ -0,0 +1,23 @@
+"""EIA Bulk Electricty Aggregate assets."""
+from dagster import asset
+
+import pudl
+
+logger = pudl.logging_helpers.get_logger(__name__)
+
+
+@asset(
+    io_manager_key="pudl_sqlite_io_manager",
+    required_resource_keys={"datastore"},
+)
+def fuel_receipts_costs_aggs_eia(context):
+    """Extract and transform EIA bulk electricity aggregates.
+
+    Returns:
+        A dictionary of DataFrames whose keys are the names of the corresponding
+        database table.
+    """
+    logger.info("Processing EIA bulk electricity aggregates.")
+    ds = context.resources.datastore
+    raw_bulk_dfs = pudl.extract.eia_bulk_elec.extract(ds)
```

Comment:
Yes, I merged the E and T steps together though they could be separate assets. @zaneselvans do you think it would be helpful to persist the raw eia API data to the database?

Reply:
It might be useful for debugging / development if we needed to go back in and change how we deal with the timeseries & metdata information in the future. But I think we can probably put it off for now and decompose it further the next time we need to mess with it.